egs/sprakbanken/s5/local/chain/compare_wer_general.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+echo $0 $*
```

Comment:
Be sure to document what arguments are appropriate for $*. Looks like it's simply the names of the exp/ directories you want to compare. You should print this help message out if $* is empty.